app/routes/git_tokens.py

```
@@ -4,340 +4,354 @@
-            user_id=payload.user_id,
-            git_hosting=payload.git_hosting,
-            token_value=encrypted_token,
-            username=user.get("login", ""),
-        )
-
-        return APIResponse.success(
-            message=constants.TOKEN_SAVED_SUCCESSFULLY, data={"id": str(git_label.id)}
-        )
-    except Exception:
-        return APIResponse.error(message=constants.GITHUB_TOKEN_SAVE_FAILED)
+	"""Handle GitHub token validation and storage"""
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)